src/services/resourceService.test.ts

```diff
@@ -102,10 +105,10 @@ describe("Resource Service", () => {
     const service = new ResourceService(sls, options);
     const deploymentString = await service.listDeployments();
     let expectedDeploymentString = "\n\nDeployments";
-    const originalTimestamp = +MockFactory.createTestTimestamp();
+    const originalTimestamp = +MockFactory.createTestTimestamp() + 4;
     let i = 0
-    for (const dep of deployments) {
-      const timestamp = originalTimestamp + i
+    for (const dep of [...deployments].reverse()) {
```

Comment:
how is this being reverse? what's the comparison operator?

Reply:
The original `deployments` object is in ascending order by timestamp. We want to make sure the deployments come back in descending order by timestamp.

The comparison operator isn't in the tests, it's in the actual code - `resourceService` line 25